### PR TITLE
Add success url

### PR DESCRIPTION
**Pull request by eraulo**
Issue: #57 
When user save page istoriaviazen, this page is going page hatam photo before It's going to home page.

### CODE REVIEW

File: map/templates/map/phototimor_form.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
         {{ form.as_table }}
       </table>
         <input type="submit" value="Save">
+        <a href="{% url 'home' %}">Skip</a>
```
Comment (marobo) [inline map/templates/map/phototimor_form.html:14]:
```suggestion
        <button type="submit">{% trans 'Save' %}</button
        <a href="{% url 'home' %}">{% trans 'Skip' %}</a> 
```

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
-</form>
+
+{% extends 'map/base.html' %}
+{% load i18n %}
+
+{% block title %}Confirm{% endblock %}
+
+{% block content %}
+    <form method="post">{% csrf_token %}
+        <p>Are you sure you want to delete "{{ object }}"?</p>
```
Comment (marobo) [inline map/templates/map/istoriaviazen_confirm_delete.html:9]: need trans tag!

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+        <input type="submit" value={% trans "Confirm" %}>
```
Comment (marobo) [inline map/templates/map/istoriaviazen_confirm_delete.html:10]: troka nee ba button not input.

File: map/templates/map/mapview.html [hunk 4]
```diff
@@ -61,6 +61,7 @@ <h3 class="trip-text">{% trans 'Our histories for our trip' %}</h3>
                         <div class="dropdown dropdown-edit-and-delete">
                             <span class="dropdown-toggle edit-and-delete-toggle" type="button" data-toggle="dropdown"></span>
                             <ul class="dropdown-menu menu-dropdown">
+                                <li class="hover-list"><a href="{% url 'photo_viazen' via.pk %}">{% trans 'Add Photo' %}</a>
```
Comment (marobo) [inline map/templates/map/mapview.html:64]:
```suggestion
                                <li class="hover-list"><a href="{% url 'photo_viazen' via.pk %}">{% trans 'Add Photo' %}</a></li>
```

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 5]
```diff
@@ -1,5 +1,13 @@
-<form method="post">{% csrf_token %}
-    <p>Are you sure you want to delete "{{ object }}"?</p>
-    <input type="submit" value="Confirm">
-    <a href="{% url 'home' %}">Cancel </a> 
-</form>
+
+{% extends 'map/base.html' %}
+{% load i18n %}
+
+{% block title %}{% trans "Confirm" %}{% endblock %}
```
Comment (PeteCoward) [inline map/templates/map/istoriaviazen_confirm_delete.html:5]:
```suggestion
{% block title %}{% trans "Confirm Delete Journey" %}{% endblock %}
```

File: map/templates/map/istoriaviazen_confirm_delete.html [hunk 6]
```diff
@@ -1,5 +1,13 @@
...
+
+{% block content %}
+    <form method="post">{% csrf_token %}
+        <p>{% trans "Are you sure you want to delete {{ object }}?" %}</p>
```
Comment (PeteCoward) [inline map/templates/map/istoriaviazen_confirm_delete.html:9]: move the `{{ object }}` outside the trans tag

Comment (PeteCoward) [reply map/templates/map/istoriaviazen_confirm_delete.html:9]: maybe use blocktrans

Comment (PeteCoward) [reply map/templates/map/istoriaviazen_confirm_delete.html:9]: https://docs.djangoproject.com/en/3.0/topics/i18n/translation#//apple_ref/Tag/trans

File: map/views.py [hunk 7]
```diff
@@ -37,14 +37,18 @@ class HatamaViazenView(CreateView):
...
 
 
 class PhotoViazenView(CreateView):
     template_name = 'map/phototimor_form.html'
     model = PhotoTimor
     fields = ['istoriaviazen', 'image']
-    success_url = reverse_lazy('home')
+
+    def get_success_url(self):
+        return reverse_lazy('photo_viazen', args = (self.object.istoriaviazen_id,))
```
Comment (eraulo) [inline map/views.py:51]: Kolega sira ita boot sira hakarak aumenta tan foto barak ka hatama foto ida ba direitamente home page.

**Pull request merged**